packages/debug/src/browser/console/debug-console-contribution.tsx

```diff
@@ -114,6 +123,50 @@ export class DebugConsoleContribution extends AbstractViewContribution<ConsoleWi
         }));
     }
 
+    protected readonly SEVERITIES = ['Errors', 'Warnings', 'Info', 'Log'];
```

Comment:
@tolusha I'm wondering, why not use the `MessageType` directly? I believe it would also remove the need to have 
`messageType + 1` and `messageType - 1` in order places.

Reply:
@vince-fugnitto 
What about `MessageType.Progress` severity? Should we use it?

Reply:
> @vince-fugnitto
> What about `MessageType.Progress` severity? Should we use it?

I believe `progress` is mainly used for notifications, I don't believe it is useful in the context of debugging types. @AlexTugarev am I correct?

Reply:
This is confusing, but at least odd.
I'd rather derive the type of a message from a common severity than the other way around.

@tolusha and @akosyakov don't you think, this is a good chance to clean it up, i.e. to introduce `Severity` type in core?

Reply:
I guess it should be something like this
https://github.com/Microsoft/vscode/blob/master/src/vs/base/common/severity.ts#L9

Reply:
It turned out we have it already
https://github.com/eclipse-theia/theia/blob/ab/severity/packages/task/src/common/problem-matcher-protocol.ts#L69